target/linux/ath79/image/Makefile

```diff
@@ -66,6 +74,28 @@ define Device/Default
 	append-rootfs | pad-rootfs | append-metadata | check-size $$$$(IMAGE_SIZE)
 endef
 
+define Device/seama
```

Comment:
I would like to see it moved to target/linux/ath79/image/generic.mk for now, as the only image using the device definition is in that file. As soon as other subtargets require the (default) device definition, it should be moved.

Reply:
OK